### ISSUE TITLE
Bug: Screen reader doesn't read `Toast` content if `Pagination` component is on page

### ISSUE THREAD

**Issue by yhafez** [dequelabs/cauldron#1436]
### Steps to reproduce

Steps to reproduce

1) Add a `Pagination`, `Loader`, and `Toast` component to the DOM.
2) Trigger the toast to load after the loader is dismissed.

### Expected Behavior

Loading content is read, then pagination content is read, then toast content is read by screen reader.

### Actual Behavior

Loading content is read Pagination content is read, but Toast component content is not read by screen reader.

### Version

6.3.1

### What browsers are you experiencing the problem on?

Chrome

**Comment by Keerthi-Penukonda** [comment]
@yhafez how can I reproduce this issue? Is there any page where we can have all `Pagination`, `Loader`, and `Toast` components?

**Note:**
I have verified in the user access page and observed that both loader and toast message content are announced by screen reader. Is this testing adequate and accurate?

![image](https://github.com/dequelabs/cauldron/assets/40262884/e2e2beb6-e6b3-4bb0-a3d4-344d6d5bd68c)

| **Label**                         | **Value** |
| ------------------------------ | ------ |
| **Server**           | _https://axe.dequelabs.com/_ |
| **Server Version**               | _4.110.0_ |
| **Tested Browser**               | _Chrome Version 122.0.6261.129 (Official Build) (64-bit)_ |
| **OS**                                  | _Windows 10_ |

**Comment by yhafez** [comment]
Hi @Keerthi-Penukonda,

This is related to this issue that you opened: https://github.com/dequelabs/walnut/issues/7940

The user management v2 page has yet to be updated to include the fix though. I'll DM you when it's been implemented for you to test.

**Comment by Keerthi-Penukonda** [comment]
Moving this ticket to done as testing of the ticket will be conducted as part of https://github.com/dequelabs/walnut/issues/7940
